src/obfl-specification.html

```diff
@@ -2759,6 +2759,20 @@ <h4 id="L21851">Example</h4>
   </tbody>
 </table>
   
+<h3 id="L2762">mark-capital-letters</h3>
+
+<p>The value of the <code>mark-capital-letters</code> attribute can either be 'true' or 'false'.
+The value 'true' indicates that the braille translator should add capital markings to the
+text contents. The value 'false' indicates the braille translator should
+<strong>NOT</strong> add capital markings text contents.</p>
+
+<p>The overall policy for marking capital letters should be determined by the
+implementing application, for example per job or in a system setting.</p>
```

Comment:
Would it be worth adding that the selected braille system has an influence too? See the example I gave of the German braille system, which normally never indicates capital letters at the beginning of nouns.

Or, should the mark-capital-letters attribute not even have an effect on that aspect?

Reply:
I think this is more of an override feature so perhaps examples of language-specific rules don't add to the specification?

Reply:
We don't need to give examples. But saying that it is determined by the implementing application is only partly true.